flight/targets/sparky/fw/pios_board.c

```diff
@@ -319,22 +218,7 @@ static void PIOS_Board_configure_hsum(const struct pios_usart_cfg *pios_usart_hs
  * 11 pulses - external HMC5883 failed
  */
 void panic(int32_t code) {
-	while(1){
-		for (int32_t i = 0; i < code; i++) {
-			PIOS_WDG_Clear();
-			PIOS_LED_Toggle(PIOS_LED_ALARM);
-			PIOS_DELAY_WaitmS(200);
-			PIOS_WDG_Clear();
-			PIOS_LED_Toggle(PIOS_LED_ALARM);
-			PIOS_DELAY_WaitmS(200);
-		}
-		PIOS_WDG_Clear();
-		PIOS_DELAY_WaitmS(200);
-		PIOS_WDG_Clear();
-		PIOS_DELAY_WaitmS(200);
-		PIOS_WDG_Clear();
-		PIOS_DELAY_WaitmS(100);
-	}
+		PIOS_HAL_Panic(PIOS_LED_ALARM, code);
```

Comment:
Totally ok with me to keep panic xonsistent. That was more a style comment.
On Oct 9, 2015 11:44 PM, "AJ Christensen" <notifications@github.com> wrote:

> In flight/targets/sparky/fw/pios_board.c
> <https://github.com/TauLabs/TauLabs/pull/1982#discussion_r41693182>:
>
> > -		for (int32_t i = 0; i < code; i++) {
> > -			PIOS_WDG_Clear();
> > -			PIOS_LED_Toggle(PIOS_LED_ALARM);
> > -			PIOS_DELAY_WaitmS(200);
> > -			PIOS_WDG_Clear();
> > -			PIOS_LED_Toggle(PIOS_LED_ALARM);
> > -			PIOS_DELAY_WaitmS(200);
> > -		}
> > -		PIOS_WDG_Clear();
> > -		PIOS_DELAY_WaitmS(200);
> > -		PIOS_WDG_Clear();
> > -		PIOS_DELAY_WaitmS(200);
> > -		PIOS_WDG_Clear();
> > -		PIOS_DELAY_WaitmS(100);
> > -	}
> > +		PIOS_HAL_Panic(PIOS_LED_ALARM, code);
>
> I didn't end up doing this. If you feel strongly about it I can update. I
> would prefer to keep the implementation consistent with the other ones so
> far, or update them all to be consistent. WDYT?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TauLabs/TauLabs/pull/1982/files#r41693182>.
>